l10n_ch_pain_credit_transfer/tests/test_ch_sct.py

```diff
@@ -6,331 +6,318 @@
         for bank_line in bank_lines:
-            self.assertEquals(bank_line.currency_id, self.eur_currency)
-            self.assertEquals(bank_line.communication_type, "isr")
+            self.assertEqual(bank_line.currency_id, self.eur_currency)
+            self.assertEqual(bank_line.communication_type, "isr")
             self.assertTrue(
                 bank_line.communication
                 in ["132000000000000000000000014", "132000000000000000000000022"]
             )
-            self.assertEquals(bank_line.partner_bank_id, invoice1.partner_bank_id)
+            self.assertEqual(bank_line.partner_bank_id, invoice1.partner_bank_id)
 
         action = self.payment_order.open2generated()
-        self.assertEquals(self.payment_order.state, "generated")
-        self.assertEquals(action["res_model"], "ir.attachment")
+        self.assertEqual(self.payment_order.state, "generated")
+        self.assertEqual(action["res_model"], "ir.attachment")
         attachment = self.attachment_model.browse(action["res_id"])
-        self.assertEquals(attachment.datas_fname[-4:], ".xml")
+        self.assertEqual(attachment.name[-4:], ".xml")
         xml_file = base64.b64decode(attachment.datas)
         xml_root = etree.fromstring(xml_file)
         # print "xml_file=", etree.tostring(xml_root, pretty_print=True)
         namespaces = xml_root.nsmap
         namespaces["p"] = xml_root.nsmap[None]
         namespaces.pop(None)
         pay_method_xpath = xml_root.xpath("//p:PmtInf/p:PmtMtd", namespaces=namespaces)
-        self.assertEquals(
+        self.assertEqual(
             namespaces["p"],
             "http://www.six-interbank-clearing.com/de/" "pain.001.001.03.ch.02.xsd",
         )
-        self.assertEquals(pay_method_xpath[0].text, "TRF")
+        self.assertEqual(pay_method_xpath[0].text, "TRF")
         sepa_xpath = xml_root.xpath(
             "//p:PmtInf/p:PmtTpInf/p:SvcLvl/p:Cd", namespaces=namespaces
         )
-        self.assertEquals(len(sepa_xpath), 0)
+        self.assertEqual(len(sepa_xpath), 0)
         local_instrument_xpath = xml_root.xpath(
             "//p:PmtInf/p:PmtTpInf/p:LclInstrm/p:Prtry", namespaces=namespaces
         )
-        self.assertEquals(local_instrument_xpath[0].text, "CH01")
+        self.assertEqual(local_instrument_xpath[0].text, "CH01")
 
         debtor_acc_xpath = xml_root.xpath(
             "//p:PmtInf/p:DbtrAcct/p:Id/p:IBAN", namespaces=namespaces
         )
-        self.assertEquals(
+        self.assertEqual(
             debtor_acc_xpath[0].text,
             self.payment_order.company_partner_bank_id.sanitized_acc_number,
         )
         self.payment_order.generated2uploaded()
-        self.assertEquals(self.payment_order.state, "uploaded")
+        self.assertEqual(self.payment_order.state, "uploaded")
         for inv in [invoice1, invoice2]:
-            self.assertEquals(inv.state, "paid")
+            self.assertEqual(inv.state, "posted")
```

Comment:
In `12.0` when you validate an invoice, you call `action_invoice_open` method of `account.invoice`.

In `14.0`, instead, you call `action_post` method of `account.move`: this changes field `state` from `Draft` to `Posted`.

[https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L139-L142](https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L139-L142)

Reply:
In `12.0` this assertion checked that the invoice had been paid, in `14.0` it should still check that the invoice has been paid independently from how the single fields have changed.

Since `inv.state` does not have the value `paid` anymore, maybe you should check another field, probably `payment_state`: https://github.com/odoo/odoo/blob/487a90a14f15ea1983ef98aba419320c1f0cc4e7/addons/account/models/account_move.py#L228